goguen/solidity.nix

```diff
@@ -20,14 +20,17 @@ pkgs.stdenv.mkDerivation {
 
   patches = [ ./solidity.patch ];
 
-  preConfigure = ''
+  configurePhase = ''
     export NIX_CFLAGS_COMPILE="-Wno-error=maybe-uninitialized $NIX_CFLAGS_COMPILE"
+    mkdir build
+    cd build
+    cmake ..
```

Comment:
the default configurephase should hande `build` for you, what happens if this part of the change is undone?

Reply:
I'm looking towards removing the initial G-ala-C implementation at some point in the future.
We can't do it for a reason right now.
In this light, I'm not sure it'd be an economic expenditure of effort to focus on making this pretty.